feature/modules/etc/smartics/camel-modules.xml

```diff
@@ -520,6 +520,34 @@
         </dependencies>
     </module>
 
+    <module name="org.apache.camel.component.consul">
+        <include artifact="org.apache.camel:camel-consul" />
+        <include artifact="com.orbitz.consul:consul-client">
+            <filter>
```

Comment:
Can these be covered by `<exports>` instead?

Reply:
Of course, if that's preferable.